pkgs/applications/science/math/sage/sagelib.nix

```diff
@@ -51,6 +52,8 @@
 , libbraiding
 , gmpy2
 , pplpy
+, sqlite # FIXME
```

Comment:
What needs to be fixed here?

Reply:
To determine where the dependencies come from. The dependency to `gaic` is explicitly mentioned in [the official document](https://wiki.sagemath.org/ReleaseTours/sage-9.2):

> The packages giacpy_sage and sage_brial have been merged into sagelib as sage.libs.giac and sage.rings.polynomial.pbori.

However, I have no idea about SQLite and Boehm GC (libgc). libgc especially *smells;* there might be a lack of build-time configuration or something. That is why I marked here.

Surely they are not clear to others, I will add some more words. Thank you for your comment!